std/portable/index.d.ts

```diff
@@ -312,57 +312,132 @@ declare interface StringConstructor {
   fromCodePoints(arr: i32[]): string;
 }
 
+declare interface String {
+  /** Returns value using relative indexing. Index may be negative */
+  at(index: i32): string;
+}
+
```

Comment:
In `portable` we are reusing the TS-provided definition of `String`. Does is "merge" with its definition and add to it, or what is the idea here?

Reply:
Yes, I think that with this method in the original interface they should merge without any conflicts

Reply:
https://www.typescriptlang.org/play?ts=4.4.0-beta#code/MYGwhgzhAEDKAuAnaBTAHvFA7AJjBiAllgObQDeAUNDdGPABTE7oBc0WArgLYBGKiAJTsISYmSq0p0RCnidEWaPAAWhCAG1m6ALoBuarQC+lE5Ragws6MUyIAZmGAo4SCoZr0muNhx78hETFSAyMgA